tests/ecosystem/upgrade/test_upgrade.py

```diff
@@ -25,6 +26,7 @@ def finalizer():
     request.addfinalizer(finalizer)
 
 
+@purple_squad
```

Comment:
@AviadP @ebenahar Which squad should be the owner of those disruptive tests executed during upgrade?

Reply:
since this is part of upgrade testing, purple squad is the most logic fit IMO